CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
+## [0.23.2] - 2020-06-23
```

Comment:
wrong year? also wrong version.

Reply:
Whoops, good catch. Should this be a minor version bump to 0.24.0?